srcpkgs/R-cran-svglite/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'R-cran-svglite'
+pkgname=R-cran-svglite
+version=2.0.0
+revision=1
+build_style=R-cran
+makedepends="R-cran-systemfonts R-cran-cpp11 libpng-devel"
+depends="R-cran-systemfonts>=1.0.0"
```

Comment:
Since it's a new package being added with this version, I don't think specifying version here is necessary.

Reply:
alright removed